gitlab/v4/objects.py

```diff
@@ -4042,6 +4062,8 @@ class Project(SaveMixin, ObjectDeleteMixin, RESTObject):
         ("variables", "ProjectVariableManager"),
         ("wikis", "ProjectWikiManager"),
         ("clusters", "ProjectClusterManager"),
+        ("additionalstatistics", "ProjectAdditionalStatisticsManager"),
```

Comment:
Why not just `statistics`, like GitLab does it? 

I see that this might be introduce confusion with `project.attributes["statistics"]`. What do you think?

Reply:
Yes, it overrides that. This is why I had to rename it.